pkg/cloudprovider/providers/aws/aws_test.go

```diff
@@ -861,7 +861,7 @@ func TestGetInstanceByNodeNameBatching(t *testing.T) {
 
 	}
 
-	instances, err := c.getInstancesByNodeNames(nodeNames)
+	instances, _ := c.getInstancesByNodeNames(nodeNames)
```

Comment:
I'd suggest to handle the error here rather than ignore it.

Reply:
I think it is ok ,because in test case , 
when err != nil , instances is empty
 and
assert.NotEmpty(t, instances) will fail .